test/Polly.Specs/RateLimit/RateLimitPolicySpecsBase.cs

```diff
@@ -11,7 +11,7 @@ public abstract class RateLimitPolicySpecsBase : RateLimitSpecsBase
         TimeSpan perTimeSpan,
         int maxBurst);
 
-    protected abstract (bool, TimeSpan) TryExecuteThroughPolicy(IRateLimitPolicy policy);
+    protected abstract (bool permitExecution, TimeSpan retryAfter) TryExecuteThroughPolicy(IRateLimitPolicy policy);
```

Comment:
We might currently have the rule for this off, but there's another rule that requires named tuples to use PascalCase (so `PermitExecution` and `RetryAfter`), so as you're adding these new it would be better to name them that way now so we don't need to rename them later.

Reply:
If I use PascaleCase I get a warning about the SA1316 rule.

Reply:
The documentation for the rule states that PascalCase is what should be used: [SA1316](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/f5f4ca9468fc29a4d34d50541afb2e496933feda/documentation/SA1316.md?plain=1#L29-L49)

Looks like the configuration is overriding the default here: https://github.com/App-vNext/Polly/blob/0064b4de2da37cc98400553834ae6a990625496b/eng/analyzers/Stylecop.json#L22

Can you remove that override as I think Pascal Case is what we should be using, provided that that change doesn't suddenly cause lots of new warnings.